unity/CITools/BuildDriver/TestTargets.cs

```diff
@@ -6,8 +6,10 @@ namespace BuildDriver;
 [Flags]
 public enum TestTargets
 {
-    None = 1 << 0,
-    Embedding = 1 << 1,
-    CoreClr = 1 << 2,
+    None = 0,
+    EmbeddingManaged = 1 << 1,
```

Comment:
nit: I think you want to start at `1 << 0` not `1 << 1` Sorry if I messed this up in my original comment. That way you start at 1 and not 2.